keras_hub/src/layers/modeling/anchor_generator.py

```diff
@@ -5,19 +5,14 @@
 
 from keras_hub.src.api_export import keras_hub_export
 
-# TODO: https://github.com/keras-team/keras-hub/issues/1965
-from keras_hub.src.bounding_box.converters import convert_format
```

Comment:
can we remove the bounding box stuff in KerasHub with this? Or do we still have usage

Reply:
Yup there should not be any usage of bounding box in KerasHub. Should I remove them in this PR itself.

Reply:
yeah, if they remove cleanly, please do!